utils/host/src/lib.rs

```diff
@@ -15,8 +15,9 @@ pub const RANGE_ELF_BUMP: &[u8] = include_bytes!("../../../elf/range-elf-bump");
 pub const RANGE_ELF_EMBEDDED: &[u8] = include_bytes!("../../../elf/range-elf-embedded");
 pub const AGGREGATION_ELF: &[u8] = include_bytes!("../../../elf/aggregation-elf");
 
-// TODO: Update to Celestia Range ELF Embedded
-pub const CELESTIA_RANGE_ELF_EMBEDDED: &[u8] = include_bytes!("../../../elf/range-elf-embedded");
+#[cfg(feature = "celestia")]
```

Comment:
There's no need to gate this here, we want it to be accessible in `config.rs`, as users should be able to get the Celestia DA verification key without passing the `--features celestia` flag.